hypothesis-python/src/hypothesis/extra/numpy.py

```diff
@@ -44,11 +46,28 @@
 from hypothesis.internal.reflection import proxies
 from hypothesis.internal.validation import check_type
 from hypothesis.strategies._internal.numbers import Real
-from hypothesis.strategies._internal.strategies import T, check_strategy
+from hypothesis.strategies._internal.strategies import Ex, T, check_strategy
 from hypothesis.strategies._internal.utils import defines_strategy
 
+
+def _try_import(fq_name: str) -> Any:
+    mod_name, _, attr_name = fq_name.rpartition(".")
```

Comment:
I think this would be a cleaner interface if it took `(module_name, attr_name)` and did an `assert "." not in attr_name` to clarify that it has to be a top-level name.

Reply:
Will do. Should it be moved to some common helpers module, or leave it where it is until there is another caller?

Reply:
I'd leave it here.